packages/api-client/src/conversation/ConversationAPI/ConversationAPI.ts

```diff
@@ -151,7 +152,10 @@ export class ConversationAPI {
   public async deleteService(conversationId: string, serviceId: string): Promise<void> {
     const config: AxiosRequestConfig = {
       method: 'delete',
-      url: `${ConversationAPI.URL.CONVERSATIONS}/${conversationId}/${ConversationAPI.URL.BOTS}/${serviceId}`,
+      url:
+        this.backendFeatures.version >= 7
```

Comment:
Should we keep it (7) as const somewhere?

Reply:
addressed here https://github.com/wireapp/wire-web-packages/pull/6715/commits/5e984b736183dfc3419c70aff9dc9713a2e30148